iothub/device/samples/readme.md

```diff
@@ -60,7 +60,7 @@ You need to clone the repository or download the sample (the one you want to try
 [d-file-upload-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/FileUploadSample
 [d-keys-rollover-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/KeysRolloverSample
 [d-import-export-devices-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/ImportExportDevicesSample
-[d-pnp-sample]: /iothub/device/samples/DigitalTwinDeviceSamples
+[d-pnp-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/PnpDeviceSamples
```

Comment:
We'll need to prep a PR for the samples to master too. This URL will be invalid for a short period of time.

Perhaps we should make it against the feature branch, and then update this file after that samples repo merge?

Reply:
Sure. I can do that.